app/controllers/stories_controller.rb

```diff
@@ -60,7 +60,7 @@ def warm_comments
   def assign_hero_html
     return if SiteConfig.campaign_hero_html_variant_name.blank?
 
-    @hero_html = HtmlVariant.relevant.select(:html).
+    @hero_html = HtmlVariant.relevant.select(:name, :html).
       find_by(group: "campaign", name: SiteConfig.campaign_hero_html_variant_name)&.html
```

Comment:
I think the build is still going to fail as the last thing this does is to select `.html`. 
I didn't notice this in the previous review

Reply:
ohhhh. got it.